codechecker_common/logger.py

```diff
@@ -182,4 +182,26 @@ def setup_logger(log_level=None, stream=None):
             if 'stream' in handler:
                 handler['stream'] = stream
 
+    # if workspace is set, we will log to a file in the workspace
+    # This is added dynamically because the declaratice config is not
+    # flexible enough, and will always create a log file in wierd locations,
+    # before we can initialize the filneame attribute, to the workspace
+    # directories
```

Comment:
```suggestion
    # If workspace is set, we will log to a file in the workspace.
    # This is added dynamically because the declarative config (config/logger.conf) is not
    # flexible enough, and will always create a log file in weird locations,
    # before we can initialize the filename attribute, to the workspace
    # directories.
```